django_test_migrations/autonames.py

```diff
@@ -13,6 +13,18 @@
 #: Settings name for this check to ignore some migrations.
 _SETTINGS_NAME: Final = 'DTM_IGNORED_MIGRATIONS'
 
+# Special key to ignore all migrations inside an app
+_IGNORE_APP_MIGRATIONS_SPECIAL_KEY: Final = '*'
+
+_IGNORED_MIGRATIONS: Set[Tuple[str, str]] = getattr(
```

Comment:
let's use [`Final`](https://docs.python.org/3/library/typing.html#typing.Final) like in typing above, because this value is not going to be reassigned